src/SFA.DAS.FindApprenticeshipTraining/InnerApi/Requests/GetProvidersByCourseRequest.cs

```diff
@@ -4,14 +4,18 @@ namespace SFA.DAS.FindApprenticeshipTraining.InnerApi.Requests
 {
     public class GetProvidersByCourseRequest : IGetApiRequest
     {
-        private int _courseId { get ; set ; }
+        private readonly double? _latitude;
+        private readonly double? _longitude;
+        private int CourseId { get ;}
```

Comment:
Is there a reason this was changed from `_courseId`?

Reply:
Naming convention, if its readonly then it should be `_courseId`. However its a Get only property at the moment so should be named `CourseId`

Having said that, i'll push up a change to make it consistent with the other properties